### PR TITLE
Adds display of ORCID to work detail page.

### LINKED ISSUE #1735: ORCID: Update item view page to accommodate display of ORCID information
Documentation and mockups: https://docs.google.com/document/d/1LXsN7yfzZCnBwHkI_L7sOni5mkecJ3t1fiyR_5BT4lQ/edit#

Once this information is entered, it will also be displayed on the item view page next to the individual's name. There is a standard from ORCID regarding how ORCIDs should be displayed; see https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier. 

The green ORCID icon can be found at https://orcid.figshare.com/articles/figure/ORCID_iD_icon_graphics/5008697.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #1735

## Why was this change made?
Per ticket

## How was this change tested?
Unit, local

image

## Which documentation and/or configurations were updated?
NA

### CODE REVIEW

File: app/components/works/orcid_component.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# typed: true
+# frozen_string_literal: true
+
+module Works
```
Comment (jcoyne) [inline app/components/works/orcid_component.rb:4]: Can you move this component into the `Works::Show` module?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: spec/components/works/detail_component_spec.rb [hunk 2]
```diff
@@ -74,6 +74,15 @@
...
   end
 
+  describe 'contributors' do
+    let(:work_version) { build_stubbed(:work_version, authors: [author]) }
+    let(:author) { build_stubbed(:person_author, orcid: 'https://orcid.org/0000-0002-1825-0097') }
+
+    it 'renders the contributor' do
+      expect(rendered.css('#contributors a').to_html).to include('https://orcid.org/0000-0002-1825-0097')
+    end
+  end
```
Comment (ndushay) [inline spec/components/works/detail_component_spec.rb:84]: would it be more complete to have also a spec for the contributor role?   And also to have a spec for a contributor without an ORCID?  And full name vs. first name and last name?  Asking for a friend.

**Review by ndushay — COMMENTED** [review]
Were we going to have another feature branch for ORCID, with a feature switch in config? Or am I misremembering? 

Making this review a comment;  do with it what you will.

File: spec/components/works/detail_component_spec.rb (hunk 2 above)
Comment (justinlittman) [reply spec/components/works/detail_component_spec.rb:84]: I'll add one for a contributor; I wasn't looking to write complete tests for the contributors section.

**Review by jcoyne — APPROVED** [review]

**Pull request merged**